doc/api/http.md

```diff
@@ -1473,7 +1473,7 @@ added: REPLACEME
 
 * {boolean}
 
-Is `true` if all data has been flushed to the underlying system.
+Becomes `true` when the [`'finish'`][] event is emitted.
```

Comment:
Does it become true before the `'finish'` event is emitted, or after? The distinction seems important to me, and I can't tell from the wording.

Reply:
It’s true before but in the same tick.

Reply:
I'd suggest 

> Is set to `true` before the [`'finish'`][] event is emitted.

Current text suggests to me the opposite.

Reply:
it becomes true **when** the event is emitted. Too me it's clear...

> Is set to true before the ['finish'][] event is emitted.

The problem with this is, "how much before"? What about:

> Is set to `true`**right** before the ['finish'][] event is emitted.

Reply:
Maybe instead of `right before`, go with `immediately before`?

```markdown
Is set to `true` immediately before the ['finish'][] event is emitted.
```

Reply:
suggestion by trott above sounds good. "I get a headache when I eat", did the headache preceed the eating?